### ISSUE TITLE
Crypto appendix - mention missing mechanisms

### ISSUE THREAD

**Issue by randomstuff** [OWASP/ASVS#2380]
The following mechanisms are not mentioned and may (?) be missing.

Stream ciphers:

* Salsa20
* XSalsa20 (useful?)
* XChaCha20 (useful?)

Block ciphers (found in the [recommended TLS ciphersuites](https://ciphersuite.info/search/?sort=sec-desc&q=TLs)):

* ARIA
* Camellia

Hash:

* Blake2 (used in [libsodium](https://doc.libsodium.org/hashing/generic_hashing), [PASETO v4](https://github.com/paseto-standard/paseto-spec/tree/master/docs/01-Protocol-Versions#version-4-sodium-modern))
* Blake3

Moreover (and for better PQC compliance), I would remove:

> Any other cipher options MUST NOT be used.
>
> [...]
>
> Any other method for key wrapping MUST NOT be used.

**Comment by danielcuthbert** [comment]
We love PR's btw ;) #justsaying

**Comment by randomstuff** [comment]
Yes, I thought it would be better to have this discussed before PR but this could directly be discussed in a PR instead…

**Comment by danielcuthbert** [comment]
@randomstuff I'd love your help making these please, if you have time?

**Comment by randomstuff** [comment]
@danielcuthbert, OK, I'll try to do that. I think I'd start by including all the things I've mentioned except maybe ARIA and Camellia which appears to quite niche in term of usage but I'm open to any feedback on this.

**Comment by randomstuff** [comment]
@unprovable, any feedback?
